src/mumble/MainWindow.cpp

```diff
@@ -1594,8 +1594,8 @@ void MainWindow::qmUser_aboutToShow() {
 		qaUserLocalMute->setEnabled(! isSelf);
 		qaUserLocalVolume->setEnabled(! isSelf);
 		qaUserLocalIgnore->setEnabled(! isSelf);
-		qaUserCommentReset->setEnabled(! p->qbaCommentHash.isEmpty() && (g.pPermissions & (ChanACL::Move | ChanACL::Write)));
-		qaUserTextureReset->setEnabled(! p->qbaTextureHash.isEmpty() && (g.pPermissions & (ChanACL::Move | ChanACL::Write)));
+		qaUserCommentReset->setEnabled(! p->qbaCommentHash.isEmpty() && (g.pPermissions & (ChanACL::ResetComment | ChanACL::Write)));
+		qaUserTextureReset->setEnabled(! p->qbaTextureHash.isEmpty() && (g.pPermissions & (ChanACL::ResetTexture | ChanACL::Write)));
```

Comment:
While this is the correct behavior for new servers, for old ones this is not true.

Thus we should check the server's version here as well and if it is less than 1.4.0, we want to keep the old behavior and only if it is 1.4.0 or higher do we want to use the new one.

Otherwise the new client is broken when connected to older servers :)